setup.py

```diff
@@ -115,6 +115,7 @@ def write_version(filename=os.path.join(*['airflow',
 dask = [
     'distributed>=1.15.2, <2'
     ]
+databricks = ['requests>=2.5.1, <3']
```

Comment:
Airflow itself depends on requests. But I put this here in case Airflow removes that dependency.